### ISSUE TITLE
Hello, here I want to test the effect of Faster-R CNN fine tuning, but do not see the weight parameter?

### ISSUE THREAD

**Issue by luoyq6** [liming-ai/AlignDet#10]
Hello, here I want to test the effect of Faster-R CNN fine tuning, but do not see the weight parameter? Is it this final model.pth file? What is the meaning of selfsup_mask-rcnn_mstrain-soft-teacher_sampler-4096_temp0.5
![image](https://github.com/liming-ai/AlignDet/assets/110521846/fa8e84e8-5e88-4cfc-a8f1-36a07b025072)

**Comment by liming-ai** [comment]
> Hello, here I want to test the effect of Faster-R CNN fine tuning, but do not see the weight parameter? Is it this final model.pth file? What is the meaning of selfsup_mask-rcnn_mstrain-soft-teacher_sampler-4096_temp0.5 ![image](https://user-images.githubusercontent.com/110521846/260341540-fa8e84e8-5e88-4cfc-a8f1-36a07b025072.png)

Hi, the Faster R-CNN and Mask R-CNN have the same pre-train weights. The only difference between them is weather adding a mask_head during fine-tuning.

**Comment by luoyq6** [comment]
ok, Thank you
